<launch>
  <!-- パラメータの設定 -->
  <param name="default_height" value="0.0" />
  <param name="leg_x" value="150.0" />
  <param name="leg_y" value="150.0" />
  
  <!-- 初期位置設定ノードの起動 -->
  <node name="initial_position_setter" pkg="dual_leg_controller" type="initial_position_setter" output="screen" />
  
  <!-- 各脚のコントローラを起動（これは既存のものを使用） -->
  <include file="$(find dual_leg_controller)/launch/hexa_leg_controller.launch" />
</launch>